README.md

```diff
@@ -1,7 +1,112 @@
+ [![Codacy Badge](https://api.codacy.com/project/badge/Grade/dc09a913cbe544dba54a21116d3f5fc7)](https://www.codacy.com/app/SpineEventEngine/core-java?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=SpineEventEngine/core-java&amp;utm_campaign=Badge_Grade) &nbsp;
 [![license](https://img.shields.io/badge/license-Apache%20License%202.0-blue.svg?style=flat)](http://www.apache.org/licenses/LICENSE-2.0)
- [![codecov.io](https://codecov.io/github/SpineEventEngine/core-java/coverage.svg?branch=master)](https://codecov.io/github/SpineEventEngine/core-java?branch=master)
- [![Codacy Badge](https://api.codacy.com/project/badge/Grade/dc09a913cbe544dba54a21116d3f5fc7)](https://www.codacy.com/app/SpineEventEngine/core-java?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=SpineEventEngine/core-java&amp;utm_campaign=Badge_Grade)
 
-The Java implementation of the framework core.
+Spine Event Engine is a Java framework for building Event Sourcing and CQRS applications that are accessed by
+clients built with JavaScript, Java Nano (Android), Objective-C, and Java.
+
+Requires Java 7 or higher.
+
+## Pre-release
+The project is under active ongoing development.At this stage, we do not recommend using the framework for production purposes.
+You are welcome to experiment and [provide your feedback][email-developers].
+
+The latest stable version is [0.8.0](https://github.com/SpineEventEngine/core-java/releases/tag/0.8.0).
+
+Please track our release announcement to be informed about the production version (1.0.0) release.  
+
+## Gradle project dependencies
+
+In order to add Spine to your project, please add the following code to your `build.gradle`:
+
+```groovy
+buildscript{
+    ext {
+        spineVersion = '0.8.0'
+        spinePluginVersion = '0.7.24-SNAPSHOT'
+
+        protobufGradlePluginVersion = '0.8.0'
+
+        spineRepository = 'http://maven.teamdev.com/repository/spine'
+        spineSnapshotsRepository = 'http://maven.teamdev.com/repository/spine-snapshots'
+    }
+    
+    // Repositories for plug-ins.
+    repositories {
+        jcenter()
+
+        // Release repository
+        maven { url = spineRepository }
+        
+        // Snapshots repository
+        maven { url = spineRepository }
+    }
+        
+    dependencies {
+        // ...
+        classpath group: 'com.google.protobuf', name:'protobuf-gradle-plugin', version: protobufGradlePluginVersion        
+        classpath group: 'org.spine3.tools', name: 'protobuf-plugin', version: spinePluginVersion
+    }
+}
+
+apply plugin: 'java'
+apply plugin: 'com.google.protobuf'
+
+repositories {
+    jcenter()
+
+    maven { url = spineRepository }
+    maven { url = spineSnapshotsRepository }
+}
+
+ext {
+    protobufVersion = '3.2.0'
+}
+
+dependencies {
+        
+    // ...
+    
+    // Protobuf run-time
+    compile group: 'com.google.protobuf', name: 'protobuf-java', version: protobufVersion
```

Comment:
I believe that we don't specify this import explicitly since we have the import for Spine `core-java`.
It's worth checking, but right now `gae-java` module does not require this dependency to run.